messages/ro.json

```
@@ -57,75 +57,75 @@
+  "omnishipping.fillDeliveryData": "Completează datele de expediere",
+  "omnishipping.calculateDelivery": "Calculează costurile de expediere",
+  "omnishipping.deliverySplitDisclaimer": "Câteva articole nu pot fi livrate la această adresă. Te rugăm să alegi un punct de ridicare pentru acestea.",
+  "omnishipping.deliverySplitDisclaimerLogged": "Câteva articole nu sunt disponibile pentru adresa ta. Te rugăm să alegi un punct de ridicare pentru acestea.",
+  "omnishipping.pickupSplitDisclaimer": "Câteva articole nu sunt disponibile pentru ridicare la această locație. Furnizează o adresă de expediere pentru a fi livrate.",
+  "omnishipping.pickupSplitDisclaimerLogged": "Câteva articole nu sunt disponibile pentru expedierea la această locație. Furnizează o adresă de expediere pentru a fi livrate.",
   "omnishipping.price": "{value, plural, =0 {gratuit} other {{formattedPrice}}}",
   "omnishipping.of": "de",
   "omnishipping.askGeolocation": "Caută o adresă cu locația ta",
   "omnishipping.cheapestOption": "Cea mai ieftină",
   "omnishipping.fastestOption": "Cea mai rapidă",
   "omnishipping.combinedOption": "Combinat",
-  "omnishipping.yourPackages": "Coletele tale",
-  "omnishipping.askGeolocationTitle": "Îți vom sugera cel mai apropiat punct de ridicare",
-  "omnishipping.askGeolocationSubtitle": "Pentru asta, avem nevoie de locația ta",
-  "omnishipping.pickupSomeItems": "Opțiuni de ridicare de la un punct",
-  "omnishipping.receiveRemainingPackages": "Primește produsele rămase",
-  "omnishipping.askGeolocationAccept": "Folosește locația curentă",
+  "omnishipping.yourPackages": "Pachete și date de livrare",
+  "omnishipping.askGeolocationTitle": "Introdu locația pentru a selecta cel mai apropiat punct de ridicare din magazin",
+  "omnishipping.askGeolocationSubtitle": "Selectează din opțiunile de mai jos",
+  "omnishipping.pickupSomeItems": "Opțiuni puncte de ridicare",
+  "omnishipping.receiveRemainingPackages": "Opțiunie de expediere pentru produsele rămase",
+  "omnishipping.askGeolocationAccept": "Folosește locația mea actuală",
   "omnishipping.askGeolocationManual": "Caută manual o adresă",
   "omnishipping.askGeolocationDenied": "Caută o adresă",
-  "omnishipping.askGeolocationDeniedTitle": "Găsește cele mai apropiate puncte de ridicare",
-  "omnishipping.askGeolocationDeniedSubtitle": "Caută o adresă pe care o frecventezi și îți vom localiza cele mai apropiate puncte de ridicare",
+  "omnishipping.askGeolocationDeniedTitle": "Găsește un punct de ridicare din magazin din apropiere",
+  "omnishipping.askGeolocationDeniedSubtitle": "Caută adresele pe care le folosești de obicei, iar noi îți vom localiza cele mai apropiate puncte de ridicare din magazin",
   "omnishipping.askGeolocationWaiting": "Așteptăm autorizația ta",
-  "omnishipping.askGeolocationInstructions": "Selectează \"Permite\" în pop-up-ul browserului",
+  "omnishipping.askGeolocationInstructions": "Selectează „Permite\" în pop-up-ul browserului",
```

Change quotation mark.
```suggestion
  "omnishipping.askGeolocationInstructions": "Selectează „Permite“ în pop-up-ul browserului",
```